test/preview/components/retrievers/test_memory_retriever.py

```diff
@@ -24,40 +26,36 @@ def mock_docs():
 class Test_MemoryRetriever(BaseTestComponent):
```

Comment:
I know it's unrelated but mixed casing is ugly as hell. 😅 

```suggestion
class TestMemoryRetriever(BaseTestComponent):
```